### PR TITLE
Add SendGrid icon

### LINKED ISSUE #11946: Request: SendGrid
### Brand Name

SendGrid

### Website

https://sendgrid.com

### Popularity Metric

Screenshot 2024-10-07 at 10 55 58

### Official Resources for Icon and Color

none

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by zcraber**
sendgrid

**Issue:** closes #11946

**Popularity metric:** [23,983](https://www.similarweb.com/website/sendgrid.com/#overview)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Why did you pick the hex value?
Picked from the icon.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -14066,6 +14066,12 @@
             "source": "https://design.sencha.com",
             "guidelines": "https://design.sencha.com/productlogo.html"
         },
+        {
+            "title": "SendGrid",
+            "hex": "55A8DD",
```
Comment (uncenter) [inline _data/simple-icons.json:14071]:
```suggestion
            "hex": "51A9E3",
```
This is the hex listed under "SendGrid Colors" for "SendGrid Blue", and the color I'm finding in the SVGs provided at the download page. Where did you find the other one?

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (zcraber) [reply _data/simple-icons.json:14071]: There's a colour version of the icon on their brand resources page: https://sendgrid.com/en-us/resource/brand

Comment (uncenter) [reply _data/simple-icons.json]: > There's a colour version of the icon on their brand resources page: [sendgrid.com/en-us/resource/brand](https://sendgrid.com/en-us/resource/brand).

The color in the SVG available for download is the same as the color listed too.

Comment (uncenter) [reply _data/simple-icons.json]: Ah, you are using the EPS file. There is an SVG available for download if you click the "Download Brand Assets" button:

CleanShot 2024-10-10 at 10 24 25

**Review by uncenter — APPROVED** [review]

**Pull request merged**